UnityProject/Assets/Scripts/Items/Others/HolyBook.cs

```diff
@@ -122,5 +122,25 @@ public IEnumerator OnSuicide(GameObject performer)
 			Chat.AddActionMsgToChat(performer, $"{performer.ExpensiveName()} farts on the holy book.");
 			performer.Player().Script.playerHealth.OnGib();
 		}
+
+		public bool WillInteract(HandActivate interaction, NetworkSide side)
+		{
+			if (interaction.PerformerPlayerScript.Mind.occupation.DisplayName != "Chaplain") return false;
+			return DefaultWillInteract.Default(interaction, side);
+		}
+
+		public void ServerPerformInteraction(HandActivate interaction)
+		{
+			if (interaction.PerformerPlayerScript.CurrentFaith == null)
```

Comment:
does this allow for more than one faith leader to exist in the round and they both having different faiths?

Reply:
Yup, you can have multiple faiths and faith leaders during the round.